samples/tfx/taxi-cab-classification-pipeline.py

```diff
@@ -121,35 +121,21 @@ def kubeflow_deploy_op(model: 'TensorFlow model', tf_server_name, step_name='dep
 def taxi_cab_classification(
     output: dsl.PipelineParam,
```

Comment:
Maybe remove the dsl.PipelineParam for output and project to get the codes cleaner?

Reply:
Done